types/light-sdk/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+         "export-just-namespace": false
```

Comment:
You can also avoid disabling this rule by removing the `export = LightSDK` line, removing the namespace declaration wrapping most of the other declarations, and adding export modifiers to everything, which is preferred.

Reply:
ok, thanks!